tables.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE `release`
+(
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `info` VARCHAR(255) NOT NULL,
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
```

Comment:
Oh yeah, I forgot to set the charset. I'll add that in, it'll be good to keep it all UTF-8. I cheekily stole this sql from when I was experimenting with propel and I didn't check it :3